lua/entities/gmod_wire_expression2/core/custom/cfc_table.lua

```diff
@@ -0,0 +1,99 @@
+E2Lib.RegisterExtension( "cfc_e2_lib", true )
+p
+e2function entity table:firstEntity()
+	if (this.size == 0) then return NULL else return this.n[ 1 ] end
+end
+e2function string table:firstString()
+	if (this.size == 0) then return NULL else return this.n[ 1 ] end
+end
+e2function number table:firstNumber()
+	if (this.size == 0) then return NULL else return this.n[ 1 ] end
```

Comment:
NULL is an entity if I'm not mistaken, this function returns a number. you should probably use 0 for its default value

Reply:
i would rather input nan or 0/0

Reply:
Done